extra/3-magic-8-ball.js

```diff
@@ -43,10 +43,77 @@
     Very doubtful.
 */
 
-// This should log "The ball has shaken!"
-// and return the answer.
+/* STEP 1 -------I turn those expected answers to arrays ------------*/
```

Comment:
The "whole magic 8 ball sequence", "checkAnswer works for `My reply is no.`", and checkAnswer works for `It is decidedly so.`" tests don't pass